uber/templates/base.html

```diff
@@ -205,7 +150,39 @@
                             </ul>
                         </li>
                     </ul>
-                    <ul class="nav navbar-nav" id="main-menu"></ul>
+                    {% block mainmenu %}
+                    <ul class="nav navbar-nav" id="main-menu">
+                        {% block mainmenu_items %}
+                        {% for menu_item in c.MENU_JSON_DECODED %}
+                            {% if menu_item.href %}
+                                <li><a href="{{ menu_item.href }}">{{ menu_item.name }}</a></li>
+                            {% elif menu_item.submenu %}
```

Comment:
What you may want to do instead is:
```
{% if not menu_item.submenu %}
{% if menu_item.href %}
// link blah
{% else %}
// disabled blah
{% endif %}
{% else %}
// submenu blah
{% endif %}
```

This is partly for readability's sake - when I first saw this I thought "wait, what if someone tries to have a top-level item with a link AND a submenu?" not realizing that that's impossible according to the class definition. Explicitly checking for the existence of the submenu before the existence of the link is slightly more verbose but I think will cause less confusion.